opm/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -231,6 +231,14 @@ namespace cpgrid
             // Make the grid.
             processEclipseFormat(g, nnc_cells, z_tolerance, false, turn_normals);
         }
+
+        {
+            const auto& ecl_actnum = ecl_grid.getACTNUM();
+            for (std::size_t g = 0; g < ecl_grid.getCartesianSize(); g++) {
+                if (ecl_actnum[g] != g.actnum[g])
```

Comment:
Are you sure this compiles? Looks like you are calling a member function on a `g` which is of type `size_t` here.

Reply:
Hehe - no this probably does not compile ... :-(